src/applications/disability-benefits/all-claims/utils/index.jsx

```diff
@@ -394,6 +394,11 @@ export const hasNewPtsdDisability = formData =>
     isDisabilityPtsd(disability.condition),
   );
 
+// NOTE: this will need to be updated or removed when we have a usecase for the
+// Additional Forms chapter beyond the new 0781 flow
+export const showAdditionalFormsChapter = formData =>
+  formData?.syncModern0781Flow === true;
```

Comment:
Won't this also depend on the selection on [[2.1] 0781 screener](https://www.figma.com/design/r3Aj9FtLFS989mlVeBsgJg/0781-Redesign?node-id=9250-79794&t=NszIlX2RqUx8XIU5-4) page? If a mental health condition IS NOT selected on that page, then wouldn't the Additional forms chapter not be presented?